code/datums/outfits/jobs/security.dm

```diff
@@ -4,8 +4,7 @@
 	l_ear = /obj/item/device/radio/headset/headset_sec
 	glasses = /obj/item/clothing/glasses/sunglasses/sechud
 	belt = /obj/item/weapon/storage/belt/holster/security
-	gloves = /obj/item/clothing/gloves/combat
```

Comment:
why no gloves? Security having gloves is canon in all of their art, including the new armor we're planning

https://static.wikia.nocookie.net/deadspace/images/4/4d/2l916xw.jpg/revision/latest/scale-to-width-down/1000?cb=20130316233827